magento/ece-tools
# MAGECLOUD-3665: Unable to install 2.2.6 with ece-tools develop after shell refactoring

### Description

Fixed bug after shell refactoring.
There are different logic in symfony/process packages between 2.x and 4.x. Version 4.x can accept `commandLine` as an array but in 2.x it has to be a string.

### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-3665

### Manual testing scenarios

1. Install magento 2.2.6 on cloud with configure post-deploy hook
2. Check that there are no fatal errors during post-deploy phase

### Contribution checklist
